.github/workflows/docker-image.yml

```diff
@@ -10,17 +10,22 @@ on:
         default: 'gravitino'
         options:
           - 'gravitino'
-          - 'gravitino-ci-hive'
-          - 'gravitino-ci-kerberos-hive'
-          - 'gravitino-ci-trino'
-          - 'gravitino-ci-doris'
-          - 'gravitino-ci-ranger'
+          - 'gravitino-ci:hive'
+          - 'gravitino-ci:kerberos-hive'
+          - 'gravitino-ci:trino'
+          - 'gravitino-ci:doris'
+          - 'gravitino-ci:ranger'
+          - 'gravitino-playground:trino'
+          - 'gravitino-playground:hive'
+          - 'gravitino-playground:ranger'
           - 'gravitino-iceberg-rest-server'
```

Comment:
Should you update this?

Reply:
It's the display name in GitHub image actions, I've already updated the final image name. Please refer to line 84.